rotkehlchen/tests/data/blockpit_transactions.csv

```diff
@@ -0,0 +1,18 @@
+Blockpit ID;Timestamp;Source Type;Source Name;Integration;Transaction Type;Outgoing Asset;Outgoing Amount;Incoming Asset;Incoming Amount;Fee Asset;Fee Amount;Transaction ID;Note;Merge ID
+1,55444E+13;06.01.2023 20:08;Chain;Polygon;Polygon Wallet Depot;Gift;META;10;;;MATIC;0.001537148341223424;0x000000000000000000000000000000000000dead;;
+1,55989E+11;08.01.2023 01:00;API;Binance;Binance Depot;Bounties;;;EUR;0.01061899;;;;Referral Kickback (sum);
+1,55983E+14;08.01.2023 12:44;API;Binance;Binance Depot;Trade;EUR;168.69217;MATIC;223.7;MATIC;0.2237;;spot;
```

Comment:
why you have scientific notation here for the id?

Reply:
That's just the way that blockpit does it I guess. See the example CSV file provided by the user here: https://github.com/rotki/rotki/issues/8147#issuecomment-2184109435

Reply:
I suspect it is the spreadsheet software of the user 😓  but we don't have anything else I guess